.github/workflows/CI.yml

```
@@ -14,36 +14,41 @@ jobs:
-      run: ./gradlew clean build -x test
+      - name: 1. checkout
+        uses: actions/checkout@v4
+
+      - name: 2. Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+
+      - name: 3. Gradle Caching
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)